Makefile

```diff
@@ -106,3 +96,18 @@ proto: install-tools ## Generate protobuf files
 .PHONY: bench-kafka
 bench-kafka:
 	go test -count 1 -run BenchmarkCompression -bench=. -benchmem ./services/streammanager/kafka/client
+
+install-tools:
+	go install github.com/golang/mock/mockgen@v1.6.0
+	go install mvdan.cc/gofumpt@latest
+	go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28.1
+	go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.2.0
+	go install gotest.tools/gotestsum@v1.10.0
+	go install golang.org/x/tools/cmd/goimports@latest
+	go install github.com/go-swagger/go-swagger/cmd/swagger@latest
+	bash ./scripts/install-golangci-lint.sh v1.55.0
+
+.PHONY: serve-swagger-ui
+serve-swagger-ui: install-tools
```

Comment:
Shouldn't you remove this recipe? It gives the following:

```
swagger validate --stop-on-error ./swagger.yaml
open ./swagger.yaml: no such file or directory
```

It should be replaced with a recipe that is compatible with the OpenAPI 3 specs and it can provide an HTML file and possibly also serve it via the browser.

Reply:
As mentioned in my previous comment this is the [html2 generator](https://openapi-generator.tech/docs/generators/html2), you can use that one.

Once you have the HTML file you can serve it via Go. If you can't find something reasonably small to do that feel free to build a cmd on your own ([it's quite easy](https://stackoverflow.com/a/39324149/828366)).